chemprop/features/features_generators.py

```diff
@@ -34,8 +37,10 @@ def get_features_generator(features_generator_name: str) -> FeaturesGenerator:
     :return: The desired features generator.
     """
     if features_generator_name not in FEATURES_GENERATOR_REGISTRY:
```

Comment:
```suggestion
    try:
        return FEATURES_GENERATOR_REGISTRY[features_generator_name]
    except KeyError:
        raise ValueError(...)
```
to avoid doing a second, unnecessary lookup